lib/WebAuthn/StaticProcessHelper.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace SimpleSAML\Module\webauthn\WebAuthn;
+
+use SimpleSAML\Auth;
+use SimpleSAML\Module;
+use SimpleSAML\Utils;
+
+class StaticProcessHelper
+{
+    public static function saveStateAndRedirect(&$state)
+    {
+        $id = Auth\State::saveState($state, 'webauthn:request');
+        $url = Module::getModuleURL('webauthn/webauthn.php');
+        Utils\HTTP::redirectTrustedURL($url, ['StateId' => $id]);
+    }
+
+    public static function prepareState($stateData, &$state)
+    {
+        $state['requestTokenModel'] = $stateData->requestTokenModel;
+        $state['webauthn:store'] = $stateData->store;
+        $state['FIDO2Tokens'] = $stateData->store->getTokenData($state['Attributes'][$stateData->usernameAttrib][0]);
+        $state['FIDO2Scope'] = $stateData->scope;
+        $state['FIDO2DerivedScope'] = $stateData->derivedScope;
+        $state['FIDO2Username'] = $state['Attributes'][$stateData->usernameAttrib][0];
+        $state['FIDO2Displayname'] = $state['Attributes'][$stateData->displaynameAttrib][0]; //NEBUDE FUNGOVAT KVŮLI DVĚMA DRUHŮM JMEN
```

Comment:
remove the comment if it is not true; otherwise add a comment in README that if standalone registration and `name2oid` are used together, then the webauthn auth proc filter has to run after `name2oid`